.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: java
+sudo: required
```

Comment:
Do you need the sudo here? If so why? Was just wondering.

Reply:
Sorry for the very late reply. I saw ur comment and could not reply back then and forgot afterwards 😅. I did not try it without sudo but I think it is needed since I found it in some tutorial. I think that you need sudo because ```services:
  - docker
``` does the same thing as ```before_install:
  - curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
  - sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
  - sudo apt-get update
  - sudo apt-get -y install docker-ce```
which needs sudo. Of course I am not sure.

Reply:
No worries, the page I just linked below also explained it. 😆